src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -826,6 +827,37 @@ result = file.getBytes()
 resultMetadata.put("content.type","image/png")
 ----
 
+=== Job Results
+
+Job Results are composed of two things: *Job Result List* and *Job Result Map*.
+Each task can contribute as for *Job Result List* as for *Job Result Map*.
+After job is finished, you can see Job Results in the Scheduler portal:
+
+image::scheduler-result-map.png[align=center]
+
+==== Job Result List
+
+Each task has its own result. However, some task results can be marked as *Task Result Added to Job Result*. In that case, this task result will be added to the *Job Result List*.
+You can mark your task in Studio:
+
+image::precious-result.png[align=center]
+
+In the Job xml schema it is available as `preciousResult`:
+
+``` {.xml}
+<task name="Init_Task" preciousResult="true" >
+```
+
+==== Job Result Map
+
+Job has key-value map associated to it, which is called *Job Result Map*. Each task can contribute to this *Job Result Map*. Consider groovy example:
```

Comment:
`Consider the following groovy example:`

Reply:
thanks